.circleci/config.yml

```diff
@@ -19,25 +19,10 @@ commands:
     steps:
       - run: sudo bash ./scripts/install-linux-deps.sh
 
-jobs:    
-  build-docker-images:
-    machine:
-      image: ubuntu-2004:current
-    steps:
-      - checkout
-      - run:
-          name: Build and push Valhalla Docker image
-          command: |
-            if [[ "${CIRCLE_BRANCH}" == "master" ]] || [[ ! -z "${CIRCLE_TAG}" ]]; then
-              BUILD_TAG="${CIRCLE_TAG:-latest}"
-              echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
-              docker build --pull --tag valhalla/valhalla:run-${BUILD_TAG} .
-              docker push valhalla/valhalla:run-${BUILD_TAG}
-            fi
-
+jobs:
   lint-build-debug:
     machine:
-      image: ubuntu-2004:current
+      image: ubuntu-2204:current
```

Comment:
maybe we also want to keep building on 20.04 to keep some lower versions of deps? osx is already pretty up-to-date testing deps versions

Reply:
nah 22.04 is fine with me! is circle on 22.04 though? it would be sad if docker didnt match CI because we would merge when circle is good on 20.04 but then docker could fail. mismatch isnt good but 22.04 is fine with me across the board if that makes sense?

Reply:
ah sorry was confused about what this was in passing, i see you are making circle match docker, this is great imho!

Reply:
> mismatch isnt good but 22.04 is fine with me across the board if that makes sense

I’m not sure it does to me tbh 😄What’s the relation between the docker workflow and the testing environment on circleci? For me they’re quite independent.

Reply:
Right, got your point. I always forget that the docker build only runs after we merged.. Makes sense.